scripts/install-dependencies.sh

```diff
@@ -71,7 +71,7 @@ if [ -z "${ANDROID_HOME}" ]; then
 fi
 
 # Install support-v4
-install_jar "com.android.support" "support-v4" "19.0.1" "${ANDROID_REPOSITORY}/com/android/support/support-v4/19.0.1/support-v4-19.0.1.jar"
+install_jar "com.android.support" "support-v4" "22.2.0" "$ANDROID_HOME/extras/android/support/v4/android-support-v4.jar"
```

Comment:
Good point, I don't know the answer right now, will see if I can find the right person to ask.